Styling/README.md

```diff
@@ -1344,6 +1347,65 @@ For example:
 
 **TODO : add note about GLSL implementations requires strict type comparisons among other things: https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/140**
 
+## Vector Data
+
+A [Vector Tile](../TileFormats/VectorData/README.md) is a collection of vector features such as points, polygons, and polylines. Vector features can be styled similar to other features with `color` and `show`. Points have several other styling options:
+
+**TODO : generate these tables from the JSON schema**
+
+The following style properties apply to geometry features in vector tiles:
+
+| Style | Type | Description | Default |
+| --- | --- | --- | --- |
+| `show` | `Boolean` | Whether or not to show the feature. | `true` |
+| `color` | `vec4` | The color of the feature. | `color('#FFFFFF')` |
+
+The following style properties apply to point features in vector tiles:
+
+| Style | Type | Description | Default |
+| --- | --- | --- | --- |
+| `show` | `Boolean` | Whether or not to show the feature. | `true` |
```

Comment:
I added `pointColor` and `labelColor` so that they can have different colors from the polygond/polylines. Should they have the same color? Should the only way to make them different from using a style that uses the metadata from the batch table?

Reply:
Maybe I am confused; what does `color` do when applied to a point feature?  Is it valid to use?  Or must `pointColor` be set instead?

Reply:
`color` isn't applied to a point. Its applied to polygons and polylines. `pointColor` is applied to points only.

Reply:
OK, so question still is can't `color` apply to a point?  So that point features "inherit" the standard properties instead of ignoring some and adding new ones?

Reply:
Updated.